backend/hubspot/hubspot.go

```diff
@@ -16,13 +16,11 @@ import (
 
 type HubspotApi struct {
 	hubspotClient hubspot.Client
-	db            *gorm.DB
```

Comment:
Ended up requiring this to be passed in as an argument to each method rather than creating a new instance. Having this in a separate instance was problematic now that we wrap a bunch of the calls for creating/updating admins/workspaces in a transaction, which this new instance would have no knowledge of.